pkg/skaffold/docker/client.go

```diff
@@ -122,11 +123,20 @@ func newMinikubeAPIClient() (client.CommonAPIClient, error) {
 		version = api.DefaultVersion
 	}
 
-	return client.NewClientWithOpts(
+	api, err := client.NewClientWithOpts(
 		client.WithHost(host),
 		client.WithVersion(version),
 		client.WithHTTPClient(httpclient),
 		client.WithHTTPHeaders(getUserAgentHeader()))
+
+	// Keep the minikube environment variables
+	var environment []string
+	for k, v := range env {
+		environment = append(environment, fmt.Sprintf("%s=%s", k, v))
+	}
+	sort.Strings(environment)
```

Comment:
Is this for testing? If so, it probably makes more sense to have it in the unit test?

Reply:
It's not only for the tests but to make debugging easier. When values are always in the same order, it's simpler